etna/ciim/constants.py

```diff
@@ -36,6 +53,31 @@ class Bucket:
     is_current: bool = False
     results: List[Any] = None
 
+    aggregations: List[str] = field(default_factory=list)
+
+    @property
```

Comment:
@TNA-Allan I don't think there's any need for this to be a property. If you want it to be, The name should probably be changed to something less method-like (e.g. `aggregations_normalised`)

Reply:
Done.